### DISCUSSION TITLE
deleteAssets in API - what is the "force" parameter doing?

### DISCUSSION THREAD

**Discussion by bmachek** [immich-app/immich#12766] [Q&A] [answered]
Hi, 

Is it bypassing trash or ignoring any constraints?

Many thanks,
Bastian

**Comment by paschun** [comment, marked as answer]
According to code, it either permanently deletes or trashes

https://github.com/immich-app/immich/blob/8309b73a02d7523e07daa571980c00724405f84b/web/src/lib/utils/actions.ts#L19-L34
